src/utils/validators.js

```diff
@@ -2,16 +2,28 @@
 // Common regexes and constants
 // ==============
 const PERMALINK_REGEX = '^(([a-z0-9]+([-][a-z0-9]+)*)+)$';
-const SOCIAL_MEDIA_REGEX_PART_1 = '^(https://)?(www.)?(';
-const SOCIAL_MEDIA_REGEX_PART_2 = '.com/)([a-zA-Z0-9_-]+(/)?)+$';
-const permalinkRegexTest = RegExp(PERMALINK_REGEX);
+const URL_REGEX_PART_1 = '^(https://)?(www.)?(';
+const URL_REGEX_PART_2 = '.com/)([a-zA-Z0-9_-]+(/)?)+$';
+const PHONE_REGEX = '^\\+65(6|8|9)[0-9]{7}$'
+const TOLLFREE_PHONE_REGEX = '^1800[0-9]{7}$'
```

Comment:
I think we might want to allow spaces in the number as well - most numbers provide this for easier viewing

Reply:
I was thinking about automatically formatting the number so that the look of the numbers are consistent across all sites - Will add this feature soon.